string offset_ptp4l_enp45s0
string offset_phc2sys_enp45s0
string offset_phc2sys_enp46s0

float64 gt_box_alphasense_driver_node_imu_hz
float64 gt_box_alphasense_driver_node_cam1_hz
float64 gt_box_alphasense_driver_node_cam2_hz
float64 gt_box_alphasense_driver_node_cam3_hz
float64 gt_box_alphasense_driver_node_cam4_hz
float64 gt_box_alphasense_driver_node_cam5_hz
float64 gt_box_hesai_packets_hz
float64 gt_box_hesai_points_hz
float64 gt_box_livox_lidar_hz
float64 gt_box_livox_imu_hz

float32 cpu_usage
string avail_memory